sros2/test/sros2/commands/security/verbs/test_generate_policy.py

```diff
@@ -28,10 +28,10 @@
 @pytest.mark.skip(reason='temporarily deactivated')
 def test_generate_policy_topics():
     with tempfile.TemporaryDirectory() as tmpdir:
-        # Create a test-specific context so that generate_policy can still init
-        context = rclpy.Context()
-        rclpy.init(context=context)
-        node = rclpy.create_node('test_generate_policy_topics_node', context=context)
+        # Create a test-specific enclave so that generate_policy can still init
+        enclave = rclpy.Context()
```

Comment:
Slightly overzealous sedding: this is actually a legit context. This comment applies throughout this file.

Reply:
When I saw that change I was like "good I should not change `rclpy.Context()` to `rclpy.Enclave()` because that would not make sense" but didn't realize that the change on the left side didn't make sense either :laughing:

Reply:
Hahaha, I'm impressed you saw it at all with what you must have been scrolling through. I would have been blind by the end of all the PRs you put together.

Reply:
I cheated :)
```
F841 local variable 'context' is assigned to but never used
        context = rclpy.Context()
        ^
```